netutils/utils.go

```diff
@@ -175,9 +172,9 @@ func ParseAlias(val string) (string, string, error) {
 	if val == "" {
 		return "", "", errors.New("empty string specified for alias")
 	}
-	arr := strings.Split(val, ":")
+	arr := strings.SplitN(val, ":", 3)
```

Comment:
why 3 and not 2 ?

Reply:
I picked `3` so that the validation below would still work (i.e., so that we can distinguish between a "valid" value and one having too many `:` in it)